### PR TITLE
response middleware for api gateway microservice

### PR DESCRIPTION
**Pull request by LijuJoseJJ**
# Description
created a response middleware and handled all response using it, for both success and failure

## Related Issue
fixes#143

## How Has This Been Tested

1. by running integration test
2. by using UI.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: api-gateway/src/middlewares/auth.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
+/* eslint-disable func-names */
```
Comment (ocgordon) [inline api-gateway/src/middlewares/auth.js]: Why is eslint disabled here?

Comment (LijuJoseJJ) [reply api-gateway/src/middlewares/auth.js]: "eslint-disable func-names" removed
and name given to anonymous callback function
please take look at recent commit

**Comment by LijuJoseJJ** [comment]
@ocgordon 
this is failing probably because you are using newer generated proof.
it will work if you run ```npm run setupAll ``` again in branch ```liju.jose/143-response-middleware-api-gateway```  from which this PR has generated

**Comment by LijuJoseJJ** [comment]
@ocgordon 
I have rebase this branch from master so, now new proof will work. Hence, no need to run ```npm run setupAll``` again in this branch

**Pull request merged**